_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "New series: From the editor's desk"
+authors: 
+  - nbower
+date: 2022-08-23
+categories: 
+  - community
+---
+
+As part of our commitment to providing complete and best-in-class documentation to the OpenSearch community, we want to ensure that you have visibility into how we approach creating bar-raising content. To that end, we will be publishing regular blog posts with tips, guidance, and best practices that will help contributors collaborate with us to produce exceptional documentation. Subjects may include technical writing, style, voice and tone, brand messaging, and other topics relevant to the production of our documentation. We hope you find this to be useful, and we would love to hear your thoughts on any of the subjects we discuss in this series. Please feel free to make your voice heard as part of the OpenSearch community through our [community meetings](https://www.meetup.com/OpenSearch/), [forum discussions](https://forum.opensearch.org/), and [GitHub repository](https://github.com/opensearch-project).
```

Comment:
Could consider ending the second sentence at "collaborate with us." as you've referenced the objective of delivering great documentation in the first sentence.